server/database/config/fakeData.sql

```diff
@@ -0,0 +1,60 @@
+BEGIN;
+
+DROP TABLE IF EXISTS admin,cohort,student,project,studentProject CASCADE;
+
+CREATE TABLE admin 
+(
+    id SERIAL PRIMARY KEY NOT NULL ,
+    username VARCHAR(100) NOT NULL ,
+    password VARCHAR(100) NOT NULL 
+);
+
+CREATE TABLE cohort 
+(
+    id SERIAL PRIMARY KEY NOT NULL ,
+    name VARCHAR(100) NOT NULL ,
+    description TEXT NOT NULL ,
+    imgUrl TEXT NOT NULL ,
+    githubLink TEXT NOT NULL 
+);
+
+CREATE TABLE student 
+(
+    id SERIAL PRIMARY KEY NOT NULL ,
+    name VARCHAR(100) NOT NULL ,
+    imgUrl TEXT NOT NULL ,
+    githubLink TEXT NOT NULL ,
+    cohortId INTEGER REFERENCES cohort(id) NOT NULL 
```

Comment:
Please add ```ON UPDATE CASCADE``` and ```ON DELETE CASCADE``` for all references in the tables so it doesn't make any errors if something updated or deleted
```suggestion
    cohortId INTEGER REFERENCES cohort(id) ON UPDATE CASCADE ON DELETE CASCADE
```